packages/delegator-e2e/test/execute-stateless7702.test.ts

```diff
@@ -472,9 +472,8 @@ test('isDeployed() returns false for addresses with code that are not delegated
   ).toBe(false);
 
   // Also test with the standalone function to show it would return false too
-  const { isValid7702Implementation } = await import(
-    '@metamask/delegation-toolkit'
-  );
+  const { actions } = await import('@metamask/delegation-toolkit');
+  const { isValid7702Implementation } = actions;
```

Comment:
this can also be `import { isValid7702Implementation } from '@metamask/delegation-toolkit/actions';`